### PR TITLE
feat(CLOUDDST-22191): add pipeline for pushing binaries to dev portal

### PR DESCRIPTION
**Pull request by swickersh**
JIRA: [CLOUDDST-22191](https://issues.redhat.com/browse/CLOUDDST-22191)

Dependent on:

1. https://github.com/konflux-ci/release-service-catalog/pull/432  (content gateway task)
2. https://github.com/konflux-ci/release-service-catalog/pull/433 (CDN task)

### CODE REVIEW

**Review by johnbieren — COMMENTED** [review]
Are you wanting to get this merged? I see a TODO in the yaml file. I don't have any critiques to make on the code that is written though

**Comment by swickersh** [comment]
> Are you wanting to get this merged? I see a TODO in the yaml file. I don't have any critiques to make on the code that is written though

Yes. The TODO was for the `extract-binaries-from-image` task to support multiple components. That's now merged. I made a few small edits since your last review to incorporate the `prepare-exodus-params` task.

File: pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml
Comment (johnbieren) [inline pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml:231]: This doesn't exist

File: pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml
Comment (johnbieren) [inline pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml:258]: This doesn't exist

**Review by johnbieren — COMMENTED** [review]
The code looks fine and I am okay with merging it. I opened two threads because two of the tasks don't exist. If you are aware of that and your goal here is to just get the pipeline merged before the tasks actually exist, I am okay with that. Otherwise we can leave this PR as is until those tasks are added in another PR. Up to you

**Comment by swickersh** [comment]
> The code looks fine and I am okay with merging it. I opened two threads because two of the tasks don't exist. If you are aware of that and your goal here is to just get the pipeline merged before the tasks actually exist, I am okay with that. Otherwise we can leave this PR as is until those tasks are added in another PR. Up to you

Yes, those PRs are still open.
I'd like to go ahead and get this pipeline merged.
This pipeline wont be invoked in the current state anyway AFAIK because we don't ship any binaries to the dev portal without signing. So this is really just to satisfy the MVP feature and in Q3 it will get a slight tweak to incorporate the signing tasks.

File: pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml
Comment (swickersh) [reply pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml:231]: Replied at top level

File: pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml
Comment (swickersh) [reply pipelines/push-binaries-to-dev-portal/push-binaries-to-dev-portal.yaml:258]: replied at top level

**Review by johnbieren — APPROVED** [review]
Ack, please rebase this and I will merge once it passes e2e

**Pull request merged**
